examples/lfric/eg17/full_example_extract/README.md

```diff
@@ -30,5 +31,37 @@ make
 The location of the PSyData wrapper library can be set with
 the environment variable ``EXTRACT_DIR``, and the location
 of the LFRic infrastructure libraries is set using
-``LFRIC_DIR``.
+``LFRIC_DIR`` - both defaulting to the versions included in
+PSyclone.
 
+## Running
+You can execute the binary ``extract`` without additional parameters:
```

Comment:
Please insert a blank line between the header and the text. Also, I would suggest indirect speech here, e.g. "The binary ``extract`` can be executed ..."

Reply:
Done.